contracts/test/TokenWithBlackList.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract TokenWithBlackList is ERC20 {
+    mapping(address => bool) _blacklist;
+
+    constructor() public ERC20("yearn.finance test token with black list", "TESTTWBL") {
+        _mint(msg.sender, 30000 * 10**18);
+    }
+
+    function blacklistUpdate(address user, bool value) public virtual {
+        _blacklist[user] = value;
+    }
+
+    function isBlackListed(address user) public view returns (bool) {
+        return _blacklist[user];
+    }
+
+    function _beforeTokenTransfer(
+        address from,
+        address to,
+        uint256 amount
+    ) internal virtual override(ERC20) {
+        require(!isBlackListed(to), "Token transfer refused. Receiver is on blacklist");
```

Comment:
Can we add this funxtionality to our test token instead? (Rename to `public mapping (address => bool) _blocked` and `_setBlocked(address, bool)`)

Reply:
No reason to create an entire new token for one testing feature

Reply:
Sure I will do it